osu.Game.Rulesets.Osu.Tests/Mods/TestSceneOsuModSpunOut.cs

```diff
@@ -48,11 +49,43 @@ public void TestSpinRateUnaffectedByMods(Type additionalModType)
                 PassCondition = () =>
                 {
                     var counter = Player.ChildrenOfType<SpinnerSpmCalculator>().SingleOrDefault();
-                    return counter != null && Precision.AlmostEquals(counter.Result.Value, 286, 1);
+                    var spinner = Player.ChildrenOfType<DrawableSpinner>().FirstOrDefault();
+
+                    if (counter == null || spinner == null)
+                        return false;
+
+                    // ignore cases where the spinner hasn't started as these lead to false-positives
+                    if (Precision.AlmostEquals(counter.Result.Value, 0, 1))
```

Comment:
Not really sure this is required. The condition is an "until" one, so this is implicit. Also if you look at the output of tests, there's still a spin-up period where the values are non-zero but also don't match the expected rate yet, which would fall through to the check below anyway.

Reply:
Yeah, it makes sense to remove this. I had written this while figuring out the calculations when my two values would sit at 0, and I actually needed this case while working on it or I would get nowhere.